### PR TITLE
[ZEPPELIN-1198][Spark Standalone] Documents for running zeppelin on production environments.

### PR DESCRIPTION
**Pull request by astroshim**
### What is this PR for?
This PR is for documentation for running zeppelin on production environments. 

### What type of PR is it?
Documentation

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1198

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

**Comment by AhyoungRyu** [comment]
@astroshim Cool! Is this PR only for Spark standalone mode? If so, why don't you edit TODO list?

**Comment by AhyoungRyu** [comment]
CI failure with below error

```
Summary
-------
Generated at: 2016-07-25T16:46:38+00:00
Notes: 3
Binaries: 152
Archives: 0
Standards: 108

Apache Licensed: 106
Generated Documents: 0

JavaDocs are generated and so license header is optional
Generated files do not required license headers

2 Unknown Licenses

*******************************

Unapproved licenses:

  scripts/docker/spark-cluster-managers/spark_standalone/Dockerfile
  scripts/docker/spark-cluster-managers/spark_standalone/entrypoint.sh

```

It will be gone if you add apache license header to each docker files :)

**Comment by astroshim** [comment]
@AhyoungRyu Thank you for review this. I will fix them.

File: docs/index.md [hunk 1]
```diff
@@ -168,6 +167,9 @@ Join to our [Mailing list](https://zeppelin.apache.org/community.html) and repor
   * [Shiro Authentication](./security/shiroauthentication.html)
   * [Notebook Authorization](./security/notebook_authorization.html)
   * [Data Source Authorization](./security/datasource_authorization.html)
+* Advanced
+  * [Apache Zeppelin on Vagrant VM](./install/virtual_machine.html)
+  * [Zeppelin on Spark Cluster Mode (Standalone)](./install/spark_cluster_mode.html#spark-standalone-mode)
```
Comment (bzz) [inline docs/index.md]: If it requires Docker, shall we say it in the title,s something  like `Apache Zeppelin on Spark Cluster Mode (Standalone via Docker)` ?

Comment (astroshim) [inline docs/index.md]: It seems your title is better. I'll change the title.

**Comment by astroshim** [comment]
@bzz ping

**Comment by bzz** [comment]
Looks great to me! Merging if there is no further discussion.

> Does this needs documentation? no

I think it does ;)

**Comment by bzz** [comment]
Added fix version and assigner to JIRA

**Pull request closed without merging**